velox/substrait/VariantToVectorConverter.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "velox/vector/ComplexVector.h"
+
+namespace facebook::velox::substrait {
+/// Create Vector from velox variants.

```

Comment:
Would you document the limitations of this function. Specifically, only scalar types are supported; VARBINARY type is not supported.

Reply:
I will do it.